### PR TITLE
fluster report: fix test results formatting

### PR DESCRIPTION
**Pull request by JenySadadia**

### CODE REVIEW

File: kcidb/templates/fluster_test.j2 [hunk 1]
```diff
@@ -40,11 +40,14 @@
             {% set statuses_counts = titles_statuses_counts.get(
                                         title, {"PASS": 0, "FAIL": 0}
                                      ) %}
+            {% set fluster_test_count_len = 5 %}
```
Comment (spbnick) [inline kcidb/templates/fluster_test.j2:43]: You can calculate the counter length like this (not tested):
```jinja
{% set max_count = 0 %}
{% for _, titles_statuses_counts in boards_titles_statuses_counts.items() %}
{% for titles, statuses_counts in titles_statuses_counts.items() %}
{% set max_count = (statuses_counts.values() | list + [set_max_count]) | max %}
{% endfor %}
{% endfor %}
{% set field_width = (max_count | string | length + 1) | string %}
```

And then you can format each line like this:
```
(
    " ✅{0: >" + field_width + "}" +
    " ❌{1: >" + field_width + "}" +
    " Σ{2: >" + field_width + "}"
).format(
    statuses_counts.get("PASS", 0),
    statuses_counts.get("FAIL", 0),
    statuses_counts.values() | sum
)
```

File: kcidb/templates/fluster_test.j2 [hunk 2]
```diff
@@ -40,11 +40,14 @@
...
             {{- indent + " " + title +
                 ":" + " " * (stats_indent|length - indent|length - title|length) +
                 "✅ " + statuses_counts.get("PASS", 0) | string +
-                "  ❌ " + statuses_counts.get("FAIL", 0) | string+
-                "  Σ " + statuses_counts.values() | sum | string +
+                " " * fluster_test_count_len +
+                "❌ " + statuses_counts.get("FAIL", 0) | string+
+                " " * fluster_test_count_len +
```
Comment (spbnick) [inline kcidb/templates/fluster_test.j2:49]: This wouldn't really solve the problem, as you're always adding the same number of spaces. They will still be unaligned, just further apart.

**Review by spbnick — CHANGES_REQUESTED** [review]

File: kcidb/templates/fluster_test.j2 (hunk 1 above)
Comment (JenySadadia) [reply kcidb/templates/fluster_test.j2:43]: The length I set was based on the possible sub-tests count of fluster tests.
I'll make it dynamic. Thanks.

Comment (spbnick) [reply kcidb/templates/fluster_test.j2:43]: Just using the dynamically-calculated field width wouldn't help. You also need alignment, which is done in the second part.

File: kcidb/templates/fluster_test.j2 [hunk 3]
```diff
@@ -34,18 +34,30 @@
...
-                "\n" -}}
+                (
+                    "✅{0: >" + field_width + "}" +
+                    " ❌{1: >" + field_width + "}" +
+                    " Σ{2: >" + field_width + "}"
+                ).format(
+                statuses_counts.get("PASS", 0),
+                statuses_counts.get("FAIL", 0),
+                statuses_counts.values() | sum
+            ) }}
```
Comment (spbnick) [inline kcidb/templates/fluster_test.j2:60]: Could you fix the formatting here (closing parenthesis not aligned with the start of the line with open parenthesis):
```suggestion
                )
            }}
```

**Review by spbnick — CHANGES_REQUESTED** [review]
LGTM, except one formatting nitpick. Thank you, Jeny!

File: kcidb/templates/fluster_test.j2 (hunk 3 above)
Comment (JenySadadia) [reply kcidb/templates/fluster_test.j2]: Fixed it. Thanks.

**Review by spbnick — APPROVED** [review]
Thank you, Jeny :heart:

**Pull request merged**